deepchem/feat/materials_featurizers.py

```diff
@@ -0,0 +1,178 @@
+"""
+Featurizers for inorganic crystals.
+"""
+
+import numpy as np
+
+from deepchem.feat import Featurizer
+from deepchem.utils import pad_array
+
+from matminer.featurizers.composition import ElementProperty
+
+from pymatgen import Composition, Structure
+from pymatgen.analysis.graphs import StructureGraph
+from pymatgen.analysis.local_env import MinimumDistanceNN
+
+
+class ChemicalFingerprint(Featurizer):
```

Comment:
Can you add documentation listings for these new featurizers in https://github.com/deepchem/deepchem/blob/master/docs/featurizers.rst?

If we do this, then the docs will automatically render the docstrings and API for these classes.